app/models/meeting_option.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class MeetingOption < ApplicationRecord
+  has_many :room_meeting_options, dependent: :restrict_with_exception
```

Comment:
I think we should add this  `has_many :rooms, through: :room_meeting_options`

Reply:
What's your thinking behind `:restrict_with_exception`

Reply:
Removing a meeting option when there's rooms that depends on it is crucial.
There was no meaning behind destroying or deleting any dependent record once we remove a meeting option but at least I wanted to prevent that from happening by restricting the operation and that also silences Rubocop xD.